AnnoyancesFilter/sections/mobile-app.txt

```diff
@@ -77,6 +77,8 @@ oceans-nadia.com##.l-footer__app
 kikkoman.co.jp##.application-banner
 napbizblog.jp##.napbizapp
 napbizblog.jp##.napbiz-app-text
+nazology.net##.js-chase[data-content-piece="Googleニュース告知"]
+nazology.net##.post-googlenews
```

Comment:
TBH I'm not sure whether this should be blocked even in Annoyances @zloyden

Reply:
@Yuki2718 why not? It's not on the header / footer.

Reply:
I meant not too obtrusive but never mind, it's anyway mobile app promo.